### PR TITLE
Add DecayLROnPlateau hook

### PR DESCRIPTION
**Pull request by blazekadam**
+ remove hooks from `cxflow_tensorflow` package root

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@
 tensorflow-tensorboard>=0.1.4
 cxflow>=0.8.1
 numpy>=1.13.1
+mock>=2.0.0
```
Comment (FloopCZ) [inline requirements.txt]: This should not be here, in Python3, the `mock` is a part of the standard library under `unittest.mock`,

File: cxflow_tensorflow/tests/hooks/decay_lr_test.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
 """
 Test module for cxflow_tensorflow.hooks.DecayLR hook.
 """
-
-import tensorflow as tf
+import mock
```
Comment (FloopCZ) [inline cxflow_tensorflow/tests/hooks/decay_lr_test.py]: This should not be here, in Python3, the `mock` is a part of the standard library under `unittest.mock`,

File: cxflow_tensorflow/hooks/decay_lr.py [hunk 3]
```diff
@@ -80,3 +80,55 @@ def after_epoch(self, **_) -> None:
 
         tf.assign(self._lr,  new_value).eval(session=self._model.session)
         logging.info('LR updated to `%s`', self._lr.eval(session=self._model.session))
+
+    def after_epoch(self, **_) -> None:
+        """Call :py:meth:`_decay_variable`."""
+        self._decay_variable()
+
+
+
```
Comment (FloopCZ) [inline cxflow_tensorflow/hooks/decay_lr.py]: Three newlines.

File: cxflow_tensorflow/hooks/decay_lr.py [hunk 4]
```diff
@@ -80,3 +80,55 @@ def after_epoch(self, **_) -> None:
...
+                  greater than the mean of last 30 ``accuracy`` values.
+
+        hooks:
+          - cxflow_tensorflow.hooks.DecayLROnPlateau:
+              long_term: 100
+              short_term: 30
+              variable: accuracy
+              objective: max
+
+
```
Comment (FloopCZ) [inline cxflow_tensorflow/hooks/decay_lr.py]: Two newlines.

File: requirements.txt (hunk 1 above)
Comment (blazekadam) [reply requirements.txt]: thanks!

**Pull request merged**